integration-test/src/lib.rs

```diff
@@ -65,19 +65,22 @@ pub fn run_node(
         config_path,
     }: RunNodeArgs,
     foundry_args: FoundryArgs,
+    use_stdio: bool,
```

Comment:
How about setting this using environment argument?
Setting `use_stdio` for each tests seems awkward.
Printing log for test is not characteristic of tests.
A programmer should be able to change the log config in runtime.